tasks/release/osx.js

```diff
@@ -6,28 +6,78 @@ var jetpack = require('fs-jetpack');
 var asar = require('asar');
 var utils = require('../utils');
 var child_process = require('child_process');
-var sign = require('electron-osx-sign')
+var sign = require('electron-osx-sign');
+var download = require('electron-download');
+var extract = require('extract-zip');
+var fs = require('fs');
+var semver = require('semver');
+var path = require('path');
 
 var projectDir;
 var releasesDir;
 var tmpDir;
+var distDir;
 var finalAppDir;
 var manifest;
 var electronVersion;
 
 var init = function () {
     projectDir = jetpack;
     tmpDir = projectDir.dir('./tmp', { empty: true });
+    distDir = tmpDir.dir('./dist', { empty: true });
     releasesDir = projectDir.dir('./releases');
     manifest = projectDir.read('app/package.json', 'json');
-    electronVersion = projectDir.read('package.json', 'json').devDependencies['electron-prebuilt'];
+    electronVersion = projectDir.read('package.json', 'json').devDependencies.electron;
+    electronVersion = semver.clean(electronVersion.replace(/\^/g, ''))
```

Comment:
oh so you would be replacing any char at the beginning which is not a number, that would do it too.